client/sandbox/react-native-expo/app/play/auth-and-context.tsx

```diff
@@ -0,0 +1,169 @@
+import {
```

Comment:
Niice! @drew-harris for the example you have for web, could we add it as a page in the nextjs sandbox too? Will make it easier to dogfood this PR